config/python.org

```diff
@@ -0,0 +1,11 @@
+* Additional Python components
+
+If you want to use the Microsoft Python language server, it's in a separate LSP package that automatically downloads the language server itself:
+
+#+BEGIN_SRC emacs-lisp
+(use-package lsp-python-ms
+  :after lsp-mode
+  :hook (python-mode . (lambda ()
+                         (require 'lsp-python-ms)
```

Comment:
Why do you `(require 'lsp-python-ms)` when we `use-package` it?

Reply:
Because the package doesn't provide any autoloads based on mode or file extension, and no commands to automatically start the server. There's nothing else possible to add as the hook, and without a hook, `python-language-server` (`pyls`) will start instead.